schemas/com.google.analytics.measurement-protocol/product_custom_metric/jsonschema/1-0-0

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Google Analytics enhanced e-commerce product custom metric entity",
+  "self": {
+    "vendor": "com.google.analytics.measurement-protocol",
+    "name": "product_custom_metric",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+
+  "type": "object",
+  "properties": {
+    "productIndex": {
+      "type": "integer",
+      "minimum": 1,
+      "maximum": 200
+    },
+    "metricIndex": {
+      "type": "integer",
+      "minimum": 1,
+      "maximum": 200
+    },
+    "value": {
+      "type": ["number", "null"]
```

Comment:
shouldn't this be an integer according to: https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#pr_cm_ ?